Invoice.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace VisualCode.Entities
+{
+    public class Invoice
+    {
+        public int IdInvoice { get; set; }
+        public decimal Amount { get; set; }
+        public Customers Customer { get; set; }
+        public Seller Seller { get; set; }
```

Comment:
As I said, is better plural for name of class of entities because, you could make a difference between a class and property with the same name.